Makefile

```diff
@@ -26,12 +26,13 @@ generate-metadata: generate-crds
 	echo "Metadata generated, please make sure you add/update fields in nginx-ingress-operator.v$(TAG).clusterserviceversion.yaml"
 
 generate-bundle:
-	-rm -rf bundle
-	mkdir bundle
-	cp deploy/crds/* bundle/
-	cp deploy/olm-catalog/nginx-ingress-operator/nginx-ingress-operator.package.yaml bundle/
-	./hack/copy_manifests.sh
-	-rm bundle.zip
-	zip -j bundle.zip bundle/*.yaml
-
-.PHONY: build
\ No newline at end of file
+	@mkdir bundle/$(TAG)
+	@cp deploy/crds/* bundle/$(TAG)
+	@cp deploy/olm-catalog/nginx-ingress-operator/nginx-ingress-operator.package.yaml bundle/
```

Comment:
Is it intentional that this line copies to `bundle` and not `bundle/$(TAG)` like the others?

Reply:
yeah it's just needed once in the root folder

Reply:
thanks, looks good